projects/1.20/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -0,0 +1,580 @@
+	"item.gobber2.gobber2_ore_deepslate": "深层戈伯矿石",
+	"item.gobber2.gobber2_ore.line1": "需要下界合金等级的镐子挖掘",
+
+	"item.gobber2.gobber2_glob": "戈伯簇",
+	"item.gobber2.gobber2_ingot": "戈伯锭",
+	"block.gobber2.gobber2_block": "戈伯块",
+	"item.gobber2.gobber2_block.line1": "方便存储戈伯锭",
+	"item.gobber2.gobber2_block.line2": "破坏时掉落完整的块",
+	"item.gobber2.gobber2_block.line3": "可以重新转换成锭",
+	"item.gobber2.gobber2_globette": "戈伯团",
+	"item.gobber2.gobber2_rod": "戈伯棒",
+	"item.gobber2.gobber2_foo": "戈伯煤炭",
+	"item.gobber2.gobber2_foo.line1": "比原版煤炭、木炭更好",
+	"item.gobber2.gobber2_foo.line2": "燃烧时间：%s",
+
+	"block.gobber2.gobber2_ore_nether": "下界戈伯矿石",
+	"item.gobber2.gobber2_ore_nether": "下界戈伯矿石",
+	"item.gobber2.gobber2_ore_nether.line1": "需要戈伯等级的镐子挖掘",
+	"item.gobber2.gobber2_glob_nether": "下界戈伯簇",
+	"item.gobber2.gobber2_globette_nether": "下界戈伯团",
+	"item.gobber2.gobber2_ingot_nether": "下界戈伯锭",
+	"block.gobber2.gobber2_block_nether": "下界戈伯块",
+	"item.gobber2.gobber2_rod_nether": "下界戈伯棒",
+	"item.gobber2.gobber2_foo_nether": "下界戈伯煤炭",
+
+	"block.gobber2.gobber2_ore_end": "末地戈伯矿石",
+	"item.gobber2.gobber2_ore_end": "末地戈伯矿石",
+	"item.gobber2.gobber2_ore_end.line1": "需要下界戈伯等级的镐子挖掘",
+	"item.gobber2.gobber2_glob_end": "末地戈伯簇",
+	"item.gobber2.gobber2_globette_end": "末地戈伯团",
+	"item.gobber2.gobber2_ingot_end": "末地戈伯锭",
+	"block.gobber2.gobber2_block_end": "末地戈伯块",
+	"item.gobber2.gobber2_rod_end": "末地戈伯棒",
+	"item.gobber2.gobber2_foo_end": "末地戈伯煤炭",
+
+	"item.gobber2.dragon_star": "龙之星",
+	"item.gobber2.dragon_elytra": "龙之鞘翅",
+	"item.gobber2.dragon_elytra.tip": "注：仅用于合成，并非真正的鞘翅",
+
+	"item.gobber2.gobber2_excavator.line1": "一把能破坏3x3区域的铲子",
+	"item.gobber2.gobber2_hammer.line1": "一把能破坏3x3区域的锤子",
+	"item.gobber2.gobber2_paxel.line1": "结合了镐、斧和锹",
+	"item.gobber2.gobber2_tree_axe.line1": "会尝试砍伐整棵树",
+	"item.gobber2.gobber2_end.unbreakable": "无法破坏的工具",
+
+	"item.gobber2.gobber2_sword": "戈伯剑",
+	"item.gobber2.gobber2_pickaxe": "戈伯镐",
+	"item.gobber2.gobber2_hammer": "戈伯锤",
+	"item.gobber2.gobber2_excavator": "戈伯铲",
+	"item.gobber2.gobber2_paxel": "戈伯多功能工具",
+	"item.gobber2.gobber2_axe": "戈伯斧",
+	"item.gobber2.gobber2_tree_axe": "戈伯树斧",
+	"item.gobber2.gobber2_hoe": "戈伯锄",
+	"item.gobber2.gobber2_shovel": "戈伯锹",
+	"item.gobber2.gobber2_bow": "戈伯弓",
+
+	"item.gobber2.gobber2_sword_nether": "下界剑",
+	"item.gobber2.gobber2_sword_nether.line1": "能更高效地获得凋灵骷髅头颅",
+	"item.gobber2.gobber2_pickaxe_nether": "下界镐",
+	"item.gobber2.gobber2_hammer_nether": "下界锤",
+	"item.gobber2.gobber2_excavator_nether": "下界铲",
+	"item.gobber2.gobber2_paxel_nether": "下界多功能工具",
+	"item.gobber2.gobber2_axe_nether": "下界斧",
+	"item.gobber2.gobber2_tree_axe_nether": "下界树斧",
+	"item.gobber2.gobber2_hoe_nether": "下界锄",
+	"item.gobber2.gobber2_shovel_nether": "下界锹",
+	"item.gobber2.gobber2_bow_nether": "下界弓",
+
+	"item.gobber2.gobber2_sword_end": "末地剑",
+	"item.gobber2.gobber2_sword_sniper": "狙击手之剑",
+	"item.gobber2.gobber2_sword_sniper.line1": "具有特殊能力的无法破坏的剑",
+	"item.gobber2.gobber2_sword_sniper.line2": "右键以发射箭",
```

```suggestion
	"item.gobber2.gobber2_sword_sniper.line2": "右击以发射箭",
```


`右键`不能作为一个完整的短句以表达动作
此类问题请全文通查并修改

---

## 已修正
- [65662f2](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/4198/commits/65662f2459d623fd635d3d206733c25e8356c1de)